_includes/configuration/resize.md

```diff
@@ -2,6 +2,11 @@
 
 This option gives you the ability to disable the resize handle or set it to resize both horizontal and vertically. The option can be true, false or the string `'both'`. `False` disables the resize, `true` enables vertical resizing only, `'both'` makes it possible to resize in both directions horizontal and vertical.
 
+When resizing is enabled, the editor can be resized by either:
+
+* Clicking and dragging the resize handle.
```

Comment:
```suggestion
* Clicking and dragging the resize handle in the bottom right-hand corner.
```
Maybe?

Reply:
It's probably not needed though and if it is, it should probably be in the first sentence for this setting.

Reply:
Making a Jira to consider moving this to the first sentence for this setting, as it's out of scope. For now, I'm going to just accept your suggestion.